aspnetcore/blazor/file-uploads.md

```diff
@@ -515,6 +518,21 @@ In Blazor Server, file data is streamed over the SignalR connection into .NET co
 
 :::zone-end
 
+## Alternative approach with Azure Files
```

Comment:
What is important to call out is the preferred approach for handling file upload scenarios. Specifically, uploading files to a storage service (Azure Blob service, Azure File service, .etc. ...) and letting the server deal with the externally stored file data as needed.
Hence, let's call it out very explicitly here, that this section represents only an example of how that can be done with Azure Files.

Reply:
The main thing about specifying Azure Files explictly over Azure Storage (Blobs) is that Blob Storage doesn't have by-container quotas. The "quota" on Azure Storage is at the account level, not the container level. That's why I was feeling out Azure Files for this suggestion. Azure doesn't have anything else like this ... that offers this full list ... so that's why I wrote the draft this way.

Reply:
I guess it could say that tho ... it could remark on both generically (Azure Storage) and then for the quota bullet point it could say that Azure Files offers that one, not Azure Blob Storage. That way, the dev doesn't go to the trouble of looking into Azure Storage if they know that they want more fine-grained automatic control of quotas.